package.json

```diff
@@ -30,10 +30,10 @@
     "build:app": "parcel build app/index.html -d dist/ --public-url '.' --no-cache",
     "build:script": "parcel build app/script.js -d dist/ --no-cache",
     "build": "npm run sync-assets && npm run build:app && npm run build:script",
-    "publish:content": "aragon apm publish minor --only-content",
+    "publish:patch": "aragon apm publish patch",
     "publish:minor": "aragon apm publish minor",
-    "publish:major": "aragon apm publish major"
+    "publish:major": "aragon apm publish major",
+    "versions": "aragon apm versions"
   },
-  "keywords": [],
-  "license": "MIT"
```

Comment:
I think the CLI should strip this key out, but it's not particularly useful anyway with the `LICENSE` present in the repo (which also gets stripped out).